std/node/util.ts

```diff
@@ -46,6 +46,16 @@ export function isRegExp(value: unknown): boolean {
   return value instanceof RegExp;
 }
 
+export function isBuffer(value: unknown): boolean {
+  return value instanceof Deno.Buffer;
```

Comment:
This is likely not what Node's isBuffer does. Deno.Buffer is something different.

Reply:
Thanks, removed for now ; 
I will do my best to bring the node buffers here .